src/collections/blog/2021-08-24-experience-gsoc-piyush-singariya/index.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: GSoC Experience @ Layer5 - Piyush Singariya
+subtitle: damn, it was good!
+date: 2020-06-23 10:30:05 -0530
+author: John Doe
+thumbnail: ./gsoc-wide.png
+category: Programs
+tags:
+  - GSoC
+  - Programs
+  - Meshery
+  - Experience
+published: true
+---
+
+import { BlogWrapper } from "../Blog.style.js";
+
+import gsoc from "./gsoc.png";
```

Comment:
@piyushsingariya I think if you've forgotten to add the .png file or have imported with the wrong file name. Could you please check